### DISCUSSION TITLE
How to quickly save any article using its URL?

### DISCUSSION THREAD

**Discussion by ketozhang** [FreshRSS/FreshRSS#5715] [Q&A] [answered]
I often use my RSS feed to also save random articles I find that come from websites that I don't subscribe to.

I know you can go to *Subscription Management > Add a feed > Type of feed source > HTML + XPath* and fill out all the information. However this takes too long as compared to what I can do in Feedly's "+ Article" button.

![image](https://github.com/FreshRSS/FreshRSS/assets/13506487/b7944f2b-62de-45c4-b0c4-63e80aefc117)

---

**Is there an equivalent feature or extension for FreshRSS**

**Comment by Frenzie** [comment, marked as answer]
The answer is probably something closer to using e.g. Wallabag or Pocket.

**Reply by ketozhang** [reply]
Ah and use subscribe to the RSS from Pocket. Yeah that would work.

**Reply by ketozhang** [reply]
That feature was removed from Pocket :/ . Wallabag still has that feature.

**Reply by Alkarex** [reply]
For now, solutions include:
* https://github.com/shaarli/Shaarli
* https://wallabag.org/

But it is such a common use-case that we might add this possibility directly into FreshRSS at some point.
